src/services/postServices.js

```diff
@@ -0,0 +1,17 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+const getAllPosts = createAsyncThunk("posts/getAllPosts",
+async ()=>{
+    try{
+        const res = await axios.get('/api/posts');
+        console.log("I was here for posts")
+        return res.data;
+    }catch(err){
+        console.log(err);
```

Comment:
I think you should remove console.log for error handling you should use createAsyncThunk api